scripts/Jenkinsfiles/bokchoy

```diff
@@ -76,6 +77,31 @@ pipeline {
                 }
             }
         }
+        stage("Combine Coverage") {
+            when {
+                // Only combine coverage during context-collection
+                environment name: 'PYTEST_CONTEXTS', value: 'true'
+            }
+            steps {
+                script {
+                    sshagent(credentials: ['jenkins-worker'], ignoreMissing: true) {
+                        checkout changelog: false, poll: false, scm: [$class: 'GitSCM', branches: [[name: git_branch]],
+                            doGenerateSubmoduleConfigurations: false, extensions: [[$class: 'CloneOption', honorRefspec: true,
+                            noTags: true, shallow: true]], submoduleCfg: [], userRemoteConfigs: [[credentialsId: 'jenkins-worker',
+                            refspec: git_refspec, url: "git@github.com:edx/${REPO_NAME}.git"]]]
+
+                        for (int i = 1; i <= 25; i++) {
```

Comment:
Having these value hard coded in multiple locations seems like it will be easy to forget when/if we reshard. What do you think about making this a variable?

Reply:
makes sense.